### PR TITLE
Added quick-guide

### PR DESCRIPTION
**Pull request by shreyasshivakumara**

### CODE REVIEW

File: _includes/quick-guide-table.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        <td markdown="span"> Instruct our Helpdesk to release your study. 
+        All registered studies are automatically placed on hold until the named submission or DAC contact instructs our Helpdesk for the study to be released.</td>
+    </tr>
+
+    </tbody>
+</table> -->
+
+
+<div class = "flex-container">
+    <div class="push">1. Checklist Requirements <div class="arrow arrow-down"></div> </div>
```
Comment (jmenglund) [inline _includes/quick-guide-table.html]: I feel that "checklist requirements" is not so easy to understand. Can we write "Is SDA the right archive?" instead?

File: _includes/quick-guide-table.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    <div >Use the Submitter Portal to register your Study, Samples, Data Access Committee (DAC) and Policy</div>
+    <div class="push">5. Encrypt and upload data <div class="arrow arrow-down"></div> </div>
+    <div >Encrypt your data files using the the instructions found <a href="https://github.com/NBISweden/EGA-SE-user-docs">here.</a> </div>
+    <div class="push">6. Register experiments and runs <div class="arrow arrow-down"></div> </div>
+    <div >Associate each data file to a registered sample and study by Linking files to samples . Details of the experimental procedure you followed must be provided.</div>
+    <div class="push">7. Finish your submission <div class="arrow arrow-down"></div> </div>
+    <div >Group your runs/analysis into datasets and link them to your new or existing DAC and policy using Submitter Portal 
+        or an XML based programmatic submission. Data request are done at a dataset level, thus files within a datasets must share release conditions.</div>
+    <div class="push">8. Release </div>
+    <div>Instruct our Helpdesk to release your study.
```
Comment (jmenglund) [inline _includes/quick-guide-table.html:70]: We probably want to change the text for this step indicating that a release date will be set by default implying that the submitter is not required to contact the helpdesk for release. I will work on a new text.

**Comment by jmenglund** [comment]
I think we can remove everything about "quick guide" (in file names and text) since this will be our main submission guide. I also noticed that the markdown and html quick guide files dit not match in content.

After this PR has been merged, I will create another with changes to the text associated  with each step.

File: _includes/quick-guide-table.html (hunk 1 above)
Comment (shreyasshivakumara) [reply _includes/quick-guide-table.html]: Done!

**Review by norling — APPROVED** [review]
I think that the commented out code should be removed, but apart from that it looks great! :)

**Review by jmenglund — APPROVED** [review]

**Pull request merged**